distribution/test/itests/test-itests-common/pom.xml

```diff
@@ -114,6 +114,21 @@
             <version>2.3.17</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.cometd.java</groupId>
+            <artifactId>cometd-java-client</artifactId>
+            <version>3.0.9</version>
```

Comment:
Is this worth making a property? Not sure if we will always be using the same version across all the cometd artifacts